apollo-router-core/src/spec/query.rs

```diff
@@ -26,7 +26,13 @@ impl Query {
     ///
     /// This will discard unrequested fields and re-order the output to match the order of the
     /// query.
-    #[tracing::instrument(skip_all, level = "trace")]
+    #[tracing::instrument(
+        skip_all,
+        fields(
+            errors = %response.errors.iter().map(std::string::ToString::to_string).collect::<Vec<_>>().join(", ")
```

Comment:
@Geal is this one worth it ?

Reply:
I do not think so, and anyway that would not be the right place to trace those errors, right?